manager/manager.go

```diff
@@ -204,6 +204,70 @@ func (oo *OneOf) EditProbeIdentificationPair(old ProbeIdentificationPair, new Pr
 	}
 }
 
+// AllOf - This selector is used to ensure that all the proves in the provided list are running.
+type AllOf struct {
+	Selectors []ProbesSelector
+}
+
+// GetProbesIdentificationPairList - Returns the list of probes that this selector activates
+func (ao *AllOf) GetProbesIdentificationPairList() []ProbeIdentificationPair {
+	var l []ProbeIdentificationPair
+	for _, selector := range ao.Selectors {
+		l = append(l, selector.GetProbesIdentificationPairList()...)
+	}
+	return l
+}
+
+// RunValidator - Ensures that the probes that were successfully activated follow the selector goal.
+// For example, see OneOf.
+func (ao *AllOf) RunValidator(manager *Manager) error {
+	var errMsg []string
```

Comment:
you could use go-multierror here

Reply:
I used it in the initial list of `ActivatedProbes` but if we use it here as well, this will mess up the bullet list ... afaik there is only one level of bullets with `go-multierror`, so if multiple depths are using it, everything will be flatten into one list, and it ends up being a mess 😭